vue-analytics.d.ts

```diff
@@ -0,0 +1,132 @@
+
+declare module 'vue-analytics' {
+  import _Vue, { PluginFunction } from 'vue';
+  import VueRouter, { Route } from 'vue-router';
+
+  interface eventFn {
+    (category: string, action?: string, label?: string, value?: number): void;
+    (options: {
+      eventCategory: string,
+      eventAction: string,
```

Comment:
Lines 10-12, these should also be marked as optional, i.e.:
```
eventAction?: string,
eventLabel?: string,
eventValue?: number
```

Reply:
@johnjcamilleri fixed in #260